src/coordinator/shard_map.cpp

```diff
@@ -75,10 +75,11 @@ std::optional<LabelId> ShardMap::InitializeNewLabel(std::string label_name, std:
   };
 
   LabelSpace label_space{
-      .schema = std::move(schema),
+      .schema = schema,
       .shards = shards,
       .replication_factor = replication_factor,
   };
+  schemas[label_id] = std::move(schema);
```

Comment:
In the long term this member can be eliminated as `label_spaces` contains the same information, right?

Reply:
Yes that's true, do you think it is valuable to do this now?

Reply:
No, not right now. Push this PR, improve on this issue later.